csrc/kernel_cache.cpp

```diff
@@ -525,6 +534,15 @@ std::vector<at::Tensor> FusionExecutorCache::runFusionWithInputs(
     }
   }
   outputs.resize(new_size);
+
+  // NOTE: This should be the last code in the method to capture all host time
+  if (isProfilerEnabled()) {
+    FusionProfiler::stop();
+  }
+  if (isProfilerPrintingEnabled()) {
+    debug() << FusionProfiler::profile();
```

Comment:
Do we need to print `std::endl` at the end?

Reply:
Every row is terminated with a `std::endl` in the `operator<<()` function.
https://github.com/NVIDIA/Fuser/blob/baaacdf36e87416ca8692b0e9fb72192e08c6e89/csrc/fusion_profiler.cpp#L535